agent/app/agent_capability_test.go

```diff
@@ -1,3 +1,4 @@
+// +build !integration
```

Comment:
I had to add `!sudo` and `!integration` since `-tags 'sudo integration'` would run tests in files without the tags `sudo` or `integration` too, which is weird.